ansible/configs/ocp4-workshop/files/repos_template.j2

```diff
@@ -1,27 +1,28 @@
 [rhel-7-server-rpms]
 name=Red Hat Enterprise Linux 7
-baseurl={{own_repo_path}}/rhel-7-server-rpms
+baseurl={{own_repo_path}}/ocp/common/rhel-7-server-rpms
```

Comment:
We will need to talk to upstream cop about the implications.

Reply:
Yeah, especially for non RHEL environments

Reply:
@jimrigsbee , I don't remember the reasoning for this change. I see 300+ uses of this in the other configs that don't follow this pattern.

Reply:
I believe I found that own_repo_path needed to just be the root of the HTML server that serves the repos. Otherwise I can't know how to create relative paths.  Env_vars did specify the ocp/common in own_repo_path in these configs and I think I had problems with another repo (EPEL?) that was not found at ocp/common/*. Let's discuss the impact here because we don't want to interfere with 300 other uses of it.